__tests__/aticles.test.js

```diff
@@ -0,0 +1,51 @@
+const app = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
```

Comment:
I would recommend returning db.end() here as otherwise the db might occasionally still hang